### PR TITLE
Some suggestions for the Introduction

### PR DESCRIPTION
**Pull request by matthiasgoergens**

### CODE REVIEW

File: Introduction.md [hunk 1]
```diff
@@ -304,4 +304,11 @@ filter :: (a -> Bool) -> [a] -> [a]
 
 ## Hoogle
 
+- Search the libraries by type
 - https://www.haskell.org/hoogle/
+
+## QuickCheck
```
Comment (charleso) [inline Introduction.md]: Yeah, definitely not what I had in mind. Anything they can't use directly on the night.

There's so many goodies but I'm worried about distractions given the time. :(

File: Introduction.md [hunk 2]
```diff
@@ -202,14 +208,7 @@ startsWith i (h : v) = i == h
 
 
 
-## Everything is a function
```
Comment (charleso) [inline Introduction.md:205]: Yeah the current file needs some love. I had plans to remove this.

File: Introduction.md (hunk 1 above)
Comment (matthiasgoergens) [inline Introduction.md]: True.  Remove QuickCheck from the introduction.

**Comment by charleso** [comment]
Thanks @matthiasgoergens. I need to give this file a good cleanup. It was previously only some quick notes, but I'm running out of time before the workshop!

**Comment by charleso** [comment]
One thing I want/need to do is reshuffle the ordering. Start with calling functions first. And then defining one without types. And then with types. Or something like that.

**Comment by matthiasgoergens** [comment]
If you are going to explain type annotations, then I think perhaps running
with scoped type variables and partial type signatures might be a good idea.

No need to explain what the pragmas do.  But they make the explanation
easier, and people can just pop in the information they know.

But it's a trade off between simpler explanation, vs having to introduce
pragmas (even if left unexplained).

**Pull request merged**